velox/core/PlanNode.h

```diff
@@ -931,7 +931,7 @@ class PartitionedOutputNode : public PlanNode {
   const RowTypePtr outputType_;
 };
 
-enum class JoinType { kInner, kLeft, kRight, kFull, kSemi, kAnti };
+enum class JoinType { kInner, kLeft, kRight, kFull, kSemi, kRightSemi, kAnti };
```

Comment:
nit: Would it make sense to rename kSemi to kLeftSemi?

Reply:
It would definitely be clearer to make it `kLeftSemi`. I will make this change in a follow-up PR to keep this PR simple.